### PR TITLE
[TEC-717] :sparkles: Added config for DynamoDB, SQS

### PR DESCRIPTION
**Pull request by RodLopezLakpa**
# Description

- [x] Se agregó al stack infra de DynamoDB.
- [x] Se agregó al stack infra de SQS.
- [x] Se quitó lambdas no ocupados.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (code improvement)

# How to test?

Please describe the steps to test. 

## Evidence

Colas desplegadas en AWS
image

Tablas creadas en AWS con SST Dev
image

Log Deployment en CloudFormation
image

## Jira

[Jira link](https://sherpawmc.atlassian.net/browse/TEC-717)

### Summary by CodeRabbit

- Refactor: Updated the name of the SST app and improved the structure of our infrastructure for better organization and clarity.
- New Feature: Introduced a new function `TruoraInfraestructure` that streamlines the creation of necessary resources, enhancing the efficiency of our system setup.
- New Feature: Added `GenerateDynamoTable` function to automate the creation of three DynamoDB tables: "Templates", "Solicitude", and "SolicitudeState", simplifying database management.
- New Feature: Implemented `GenerateSQS` function to automate the creation of two SQS queues, improving message handling and system communication.
- Documentation: Included a new pull request template to standardize the description of changes, testing instructions, and Jira integration, facilitating better team collaboration.
- Style: Minor formatting adjustment in `.gitignore` file for improved readability.

### CODE REVIEW

File: stacks/MyStack.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { StackContext } from "sst/constructs";
+
+import { GenerateDynamoTable } from "./dynamodb";
+import { GenerateSQS } from "./sqs";
+
+export function TruoraInfraestructure({ stack }: StackContext) {
+  GenerateDynamoTable(stack);
+  GenerateSQS(stack);
+  return {};
```
Comment (phu154) [inline stacks/MyStack.ts:9]: no es necesario el return  {}

File: stacks/sqs/index.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { Queue, Stack } from "sst/constructs";
+
+export function GenerateSQS(stack: Stack) {
+  new Queue(stack, "truora-integration-main");
+  new Queue(stack, "truora-integration-dlq");
+}
```
[the remark this answers is not in the document]
Comment (RodLopezLakpa) [reply stacks/sqs/index.ts]: Done CodeRabbit!

File: stacks/MyStack.ts (hunk 1 above)
Comment (RodLopezLakpa) [reply stacks/MyStack.ts]: Done

File: stacks/sqs/index.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Queue, Stack } from "sst/constructs";
+
+export function GenerateSQS(stack: Stack) {
+  const mainQueue = new Queue(stack, "truora-integration-main");
+  const dlqQueue = new Queue(stack, "truora-integration-dlq");
+  return { mainQueue, dlqQueue };
+}
```
[the remark this answers is not in the document]
Comment (RodLopezLakpa) [reply stacks/sqs/index.ts:7]: It's not necessary, sst handle errors throughout CloudFormation actions

**Review by phu154 — APPROVED** [review]

**Pull request merged**